plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/style.scss

```diff
@@ -83,17 +87,19 @@ $border-width: 1px;
 		}
 	}
 
-	.wc-block-components-express-payment__title {
+	.wc-block-components-title.wc-block-components-express-payment__title {
 		flex-grow: 0;
 		transform: translateY(-50%);
+		@include font-size(smaller);
```

Comment:
I think this gets too small, even if you consider the work that I am doing to make the font sizes smaller throughout checkout. With the [purple theme](https://github.com/woocommerce/woo-themes/tree/trunk/purple) it gets to 12px for me, whereas we should aim for 14px according to fpl1YUEIW7suBFO09qymA2-fi-2158_4831. You can see my work in progress (that also touched express payments area) in https://github.com/woocommerce/woocommerce/pull/59518.